src/ShopifyApp/resources/views/layouts/default.blade.php

```diff
@@ -34,4 +34,4 @@
 
         @yield('scripts')
     </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
This was added without me noticing, but isn't it needed?

Reply:
Thats fine!